ops/ansible/roles/postmaster_deploy/handlers/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: restart nginx
+  sudo: yes
+  service: name=nginx state=restarted
+
+- name: validate nginx configuration
+  sudo: yes
+  command: nginx -t -c /etc/nginx/nginx.conf
+  changed_when: False
+
+- name: reload nginx
+  sudo: yes
+  service: name=nginx state=reloaded
+
+- name: reload uwsgi
+  sudo: yes
+  service: name=uwsgi state=reloaded
+
+- name: reload apache
+  sudo: yes
+  service: name=apache2 state=reloaded
+
+- name: restart apache
+  sudo: yes
+  service: name=nginx state=restarted
+
+- name: restart uwsgi
+  sudo: yes
+  service: name=apache2 state=restarted
+
+- name: reload supervisor
+  sudo: yes
+  service: name=supervisord state=reloaded
+
+- name: restart supervisor
+  sudo: yes
+  service: name=supervisord state=restarted
+
+- name: update supervisor
```

Comment:
I tend to disagree because we should try to have the least amount of code possible to make things more manageable. In this instance it isn't bad, but I foresee us ever using supervisor.